js/controller.js

```diff
@@ -1390,6 +1410,27 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.renderSkin();
     },
 
+    getAdRemainingTime: function() {
```

Comment:
Can we add a jsdoc for this function? I understand some other functions don't have jsdocs, but we'd like to stay on top of this moving forward.